### PR TITLE
feat: implemented rapid increase decrease of seekbar value

### LINKED ISSUE #1149: no rapid increase in up/down button in wave generator
**Actual Behaviour**

There is no way to rapidly increase the value of the seekbar using up and down buttons

**Expected Behaviour**

when the up and down buttons are long pressed the value of seekbar should rapidly increase

**Steps to reproduce it**

Wavegenerator->Implement rapid increase in up down buttons

**Would you like to work on the issue?**

Yes

### PR DESCRIPTION
**Pull request by Avjeet**
Fixes #1149 
**Changes**: implemented rapid increase in value of seekbar by long pressing up and down buttons.

**Screenshot/s for the changes**: 
ezgif com-video-to-gif 11

**Checklist**: [Please tick following check boxes with `[x]` if the respective task is completed]
- [ ] I have used resources from `strings.xml`, `dimens.xml` and `colors.xml` without hard-coding them
- [x] No modifications done at the end of resource files `strings.xml`, `dimens.xml` or `colors.xml`
- [x] I have reformatted code in every file included in this PR [<kbd>CTRL</kbd>+<kbd>ALT</kbd>+<kbd>L</kbd>]
- [x] My code does not contain any extra lines or extra spaces
- [x] I have requested reviews from other members

**APK for testing**:
[app-debug.zip](https://github.com/fossasia/pslab-android/files/2142791/app-debug.zip)

### CODE REVIEW

File: app/src/main/java/org/fossasia/pslab/activity/WaveGeneratorActivity.java [hunk 1]
```diff
@@ -801,5 +795,104 @@ public boolean onTouchEvent(MotionEvent event) {
...
+     */
+    @SuppressLint("ClickableViewAccessibility")
+    private void monitorLongClicks(ImageButton up, ImageButton down) {
+        up.setOnTouchListener(new View.OnTouchListener() {
+            @Override
+            public boolean onTouch(View view, MotionEvent motionEvent) {
+                view.onTouchEvent(motionEvent);
+                if (motionEvent.getAction() == MotionEvent.ACTION_UP) {
+                    stopCounter();
+
```
Comment (CloudyPadmal) [inline app/src/main/java/org/fossasia/pslab/activity/WaveGeneratorActivity.java]: Nice! @Avjeet shall we remove this line?

File: app/src/main/java/org/fossasia/pslab/activity/WaveGeneratorActivity.java [hunk 2]
```diff
@@ -1,6 +1,7 @@
 package org.fossasia.pslab.activity;
```
Comment (harsh-2711) [inline app/src/main/java/org/fossasia/pslab/activity/WaveGeneratorActivity.java]: Can you please remove this redundant lines? Else everything is good :+1:

File: app/src/main/java/org/fossasia/pslab/activity/WaveGeneratorActivity.java [hunk 3]
```diff
@@ -143,6 +145,9 @@
 
     private int leastCount, seekMax, seekMin;
     private String unit;
+    private Timer waveGenCounter;
+    private Handler powerHandler = new Handler();
```
Comment (CloudyPadmal) [inline app/src/main/java/org/fossasia/pslab/activity/WaveGeneratorActivity.java]: I meant in all places related to wave generator. This one has to be waveGenHandler or something.

**Comment by Avjeet** [comment]
@harsh-2711 @CloudyPadmal done with the changes

File: app/src/main/java/org/fossasia/pslab/activity/WaveGeneratorActivity.java (hunk 3 above)
Comment (Vikum94) [reply app/src/main/java/org/fossasia/pslab/activity/WaveGeneratorActivity.java]: Will approve after this is changed

**Pull request merged**